steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,51 @@
+
+I could easily have walked away from this team and the role but I choose to stay mainly because I love a challenge, no matter how impossible it seems at first.
+
+As I got to know the people on the team, over the first several months, I put together a list of 'things' that were either missing or just seemed wrong. And so it was that before the end of my probation period I presented my manager with a three pronged approach that would allow us to deliver features transparently, at pace, and with reliability - all within 12 months.
+
+My manager was skeptical of the deadline - not because of my capability (because I'd done this exact thing in a previous role) but because of the people within the team. However I got the buy-in because according to the service roadmap the future changes were going to be significant and the business were well aware of the shortcomings of the team.
+
+So began my journey of unofficial Quality Coach.
+
+The team consisted of a bunch of developers that had lots of experience of writing all the code and then throwing it over the fence, aka they only knew how to deliver changes within a Waterfall process. My challenge was to show them, educate them, and encourage them to take on board the philosophy of Agile and the Scrum methodology in particular. I should add there was a team manager but he was split across several teams and didn't have the time or experience of transforming a team to Agile.
+
+The second part of my approach focused on integrating Test Automation across the soon to be Agile/Scrum delivery lifecycle that the team would adopt. Experience had taught me a big lesson that not getting Test Automation embedded within the entire SDLC would stop all future progress to becoming an agile delivery team. Coupled with this there was the other elephant in the room 'testing is a shared team responsibility' that needed to be tackled quickly and effectively.
+
+The third part of my approach focused on the technologies and tooling for the team and I had assumed the developers would be excited at using shiny new tools, such as: Jenkins, SonarQube, Continuous Delivery, the AWS platform, and so on. I wasn't wrong in my assumption they couldn't get enough, they took all the training courses, all of them completed at least 2 levels of the AWS certification path. All that was missing was how they could create and make use of a Jenkins pipeline and that's where I could step in to help both with the design and initial implementation.
```

Comment:
focused is fine but more American/Australian  but focussed is the British version.

Reply:
Good spot - fixed